ui/test/callWithMaskHandling.test.ts

```diff
@@ -8,9 +8,11 @@ import { parseError } from '../src/parseError';
 import { assertThrowsAsync } from './assertThrowsAsync';
 
 // tslint:disable: align
+// tslint:disable: max-func-body-length
 suite("callWithMaskHandling Tests", () => {
     const credentials: string = 'scHQERrAlXSmlCeN1mrhDzsHWeDz2XZt5R343HgCNmxS0xlswcaA2Cowflda';
     const credentialsSpecialChars: string = 'sc()HQ*E+RrAlXSm[CeN1$$$rhDz^^HWeDz2X[t5R343HgCN.xS0x]swc|A2CÑwf¬da';
+    const credentialsWithReservedChars: string = '$scHQERrAlXSmlCeN1mrhDzsHWeDz2XZt5R343HgCNmxS0xlswcaA2Cowflda';
```

Comment:
Lol `credentialsWithReservedChars` feels like a lame version of `credentialsSpecialChars`

Reply:
I know haha. I wanted to make it clear that these weren't just your regular special chars however. Because I believe that reserved chars in URIs are a different set of characters than special characters. I'm gonna keep it in there.